### PR TITLE
update docker hub init

### PR DESCRIPTION
**Pull request by Jinyu-W**
# Description

## Linked issue(s)/Pull request(s)

- [issue_number](issue_link)

## Type of Change

- [ ] Non-breaking bug fix
- [ ] Breaking bug fix
- [ ] New feature
- [ ] Test
- [ ] Doc update
- [ ] Docker update

## Related Component

- [ ] Simulation toolkit
- [ ] RL toolkit
- [ ] Distributed toolkit

## Has Been Tested

- OS:
  - [ ] Windows
  - [ ] Mac OS
  - [ ] Linux
- Python version:
  - [ ] 3.6
  - [ ] 3.7
- Key information snapshot(s):

## Needs Follow Up Actions

- [ ] New release package
- [ ] New docker image

## Checklist

- [ ] Add/update the related comments
- [ ] Add/update the related tests
- [ ] Add/update the related documentations
- [ ] Update the dependent downstream modules usage

### CODE REVIEW

File: playground.md [hunk 1]
```diff
@@ -1,18 +1,21 @@
...
 | Service           | Description                                                | Host                       |
 |-------------------|------------------------------------------------------------|----------------------------|
 | `Redis Commander` | Redis Web GUI.                                             | http://127.0.0.1:40009     |
-| `Read the Docs`   | Local host docs.                                           | http://127.0.0.1:40010     |
-| `Jupyter Lab`     | Jupyter lab with MARO environment, examples, notebooks.    | http://127.0.0.1:40011     |
-*(Remember change ports, if you used different ports mapping.)*
+| `Jupyter Lab`     | Jupyter lab with MARO environment, examples, notebooks.    | http://127.0.0.1:40010     |
 
-## Major materials in root folder
+*(Remember to change ports, if you use different ports mapping.)*
```
Comment (ysqyang) [inline playground.md:12]: Remember to change ports, if you use different ports mapping.
->
Remember to change ports if you use a different port mapping.

File: maro/simulator/scenarios/cim/common.py [hunk 2]
```diff
@@ -25,10 +25,11 @@ class Action:
     Args:
         vessel_idx (int): Which vessel will take action.
         port_idx (int): Which port will take action.
-        quantity (int): How many containers can be moved from vessel to port (negative in reverse).
+        action_type (ActinType): Whether the action is a Load or a Discharge.
```
Comment (ysqyang) [inline maro/simulator/scenarios/cim/common.py:28]: ActinType -> ActionType?

File: playground.md (hunk 1 above)
Comment (Jinyu-W) [reply playground.md]: done

File: playground.md [hunk 3]
```diff
@@ -1,18 +1,21 @@
 # Playground
-Playground provides a standalone MARO running environment, it also includes examples of some predefined scenarios and quick-start tutorial.
 
-## Main service in playground
+The playground provides a standalone MARO running environment, it also includes examples of some predefined scenarios and quick-start tutorials.
```
Comment (ysqyang) [inline playground.md:3]: The playground provides a standalone MARO environment along with several predefined scenarios and quick-start tutorials.

File: docs/source/installation/playground.rst [hunk 4]
```diff
@@ -54,15 +51,12 @@ Major Services in Playground
...
-     - Local host docs.
-     - http://127.0.0.1:40010
    * - ``Jupyter Lab``
      - Jupyter lab with MARO environment, examples, notebooks.
-     - http://127.0.0.1:40011
+     - http://127.0.0.1:40010
 
 
-*(If you use other port mapping, remember to change the port number.)*
+*(Remember to change ports, if you use different ports mapping.)*
```
Comment (ysqyang) [inline docs/source/installation/playground.rst]: Remember to change ports, if you use different ports mapping.
->
Remember to change ports if a different port mapping is used.

**Comment by ysqyang** [comment]
The PR looks fine. I have no more comments.

File: maro/simulator/scenarios/cim/common.py (hunk 2 above)
Comment (Jinyu-W) [reply maro/simulator/scenarios/cim/common.py:28]: yes!

File: playground.md (hunk 3 above)
Comment (Jinyu-W) [reply playground.md:3]: how about `predefined scenario examples and ...` ?

File: docs/source/installation/playground.rst (hunk 4 above)
Comment (Jinyu-W) [reply docs/source/installation/playground.rst]: done

File: playground.md (hunk 3 above)
Comment (Jinyu-W) [reply playground.md]: done

File: maro/simulator/scenarios/cim/common.py (hunk 2 above)
Comment (Jinyu-W) [reply maro/simulator/scenarios/cim/common.py]: done

**Review by ysqyang — APPROVED** [review]

**Pull request merged**